pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -152,13 +152,13 @@
             %img#special-guest-stars{src: "images/homepage/codebreak_specialguests.png", style: "width: 300px; padding-bottom: 10px; padding-top: 30px; max-width: 100%"}
           .headshots{style: "text-align: center"}
             .guest-1{style: "display: inline-block"}
-              %img{src: "/images/fill-200/homepage/codebreak_keegan_michael_key.jpg", style: "border-radius: 50%; width: 150px; height: 150px;"}
-              %h3{style: "font-weight: 900; color:#191919; margin-top: 10px; margin-bottom: 6px"} Keegan-Michael Key
+              %img{src: "/images/fill-200/homepage/codebreak_sal_khan.jpg", style: "border-radius: 50%; width: 150px; height: 150px;"}
+              %h3{style: "font-weight: 900; color:#191919; margin-top: 10px; margin-bottom: 6px"} Sal Kahn
```

Comment:
typo: Khan

Reply:
that's why there's a review process.  🤦‍♀️thank you!